prow/config.yaml

```diff
@@ -83,3 +83,7 @@ deck:
         required_files:
           - artifacts/junit.*\.xml
     announcement: "The old job viewer, Gubernator, has been deprecated in favour of this page, Spyglass.{{if .ArtifactPath}} For now, the old page is <a href='https://gubernator-internal.googleplex.com/build/{{.ArtifactPath}}'>still available</a>.{{end}} Please send feedback to osp-engprod@google.com."
+    rerun_auth_config:
+      github_team_slugs:
+      - org: GoogleCloudPlatform
+        slug: googlers
```

Comment:
We'll probably want to set up ghproxy for this cluster since listing the member of this team currently costs 25 API ratelimit tokens. That being said, the cache entry would become invalid pretty quickly as I suspect that team has high churn and this bot token isn't heavily utilized.

Reply:
Good idea. Adding `ghproxy` can only aid in minimizing token usage, even if the cache entry does get invalidated. I can add a ghproxy deployment for this instance, #101.

Reply:
ghproxy will be added in #128